### DISCUSSION TITLE
How to make Radial progress compatible with NextJs(React)

### DISCUSSION THREAD

**Discussion by Redtef** [saadeghi/daisyui#880] [Q&A] [answered]
Hey Everyone, this is my first post in a disucussions tabs in Github so apologies if my message isn't well formatted. 
I have started learning for a simple hobby project  NextJs with Tailwind and came across DaisyUi , the components are great and easy to use except when using Radial progess.

The example given in the docs is the following : 
`<div class="radial-progress" style="--value:70;">70%</div>`
I do know that in React we should change class to className , but what about the style proprety , it wouldn't work without the `--value:70` and NextJs gives the following error 

<img width="701" alt="image" src="https://user-images.githubusercontent.com/19893917/174911257-be3ac681-8f17-46cd-8949-fff6ad0ba7e8.png">
From the search i have done , i couldn't find an example of this component being used in a nextJs project.

**Comment by CharcoalStyles** [comment, marked as answer]
Here's a little thing I mocked up from the DaisyUI StackBlitz demo.
https://stackblitz.com/edit/daisyui-nextjs-c212xy?file=pages/index.js

Basically you need to change `class` to `className` and put the `style` into React style object.

**Reply by Redtef** [reply]
Thank you !! That example was exactly what i needed.
I do realise now that solution was in the error displayed , Thankyou for your help!
